### ISSUE TITLE
Impact Interview Team Meeting

### ISSUE THREAD

**Issue by ebele-oputa** [hackforla/open-community-survey#11]
### Dependency
#40 and #20

### Overview
We need to gather all of the internal stakeholders for impact interviews so that we can interview external stakeholders one time.

### Action Items
- [x] Introduce Chianta to Rosaline, Fundraising team and Civic Opportunity team

**Comment by ebele-oputa** [comment]
@ExperimentsInHonesty introduced Chianta via slack to Alex Lee (Civic Opportunity), Ali Kiani (Citizen Engagement PM), Chelsea Beck and Melissa Mae Cruz (Fundraising team), Mike Chiusano (Marketing team), Rosalyn Broddie (HackforLA.org website team)

**Comment by ebele-oputa** [comment]
@ExperimentsInHonesty reach out to Ali and get back to team

**Comment by ExperimentsInHonesty** [comment]
Ali will be out for the next few months.  @csdorsey lets make a three way appt with Mina, you and I and figure out what our next steps are.

**Comment by ebele-oputa** [comment]
Update from @csdorsey: I met with Mina regarding other UX research projects that have been undertaken. I also reached out to folks who've done projects with DONE or Neighborhood Councils to see if they did any research projects, and I'm awaiting their answers.

**Comment by csdorsey** [comment]
Hi @ebele-oputa . I gave the above update on July 19th. Out of the 3 people I contacted, 2 responded back. I am now waiting on feedback from additional contacts that were provided to me on July 26th.

**Comment by ebele-oputa** [comment]
@csdorsey Please provide update:
1. Progress
2. Blockers
3. Availability
4. ETA

**Comment by csdorsey** [comment]
1. Alex Lee responded and said that they would be interested in data for efficacy of projects and would like to be kept in the loop on future research. Chelsey Beck, Melissa Mae Cruz, and Mike Chuisano never responded back. Before this group, I contacted every PM of a project that was working with DONE or Neighborhood to ask if they had done any interviews with the two groups. I contacted Sarah Nabelsi from the Access the Data project who stated that they hadn't done any interviews with DONE. I contacted Bryon Hart, PM of Engage project, who said they had done some interviews, but it was on agenda creation. I contacted Abhinav Bommakanti, who was listed as the PM for Lucky Parking (probably was outdated since his name has since been removed), but I never got a response from him.
2. Not sure what you mean by blockers @ebele-oputa . Can you explain?
3. My availability will end on Friday, August 13, 2021.
4. You need my ETA for what event?

**Comment by ebele-oputa** [comment]
Thanks @csdorsey. Blockers and ETA are no longer relevant since you are unavailable. Can you schedule a handover meeting with @kvnw2020  before the end of the week? Thanks

**Comment by ebele-oputa** [comment]
> Update from @csdorsey: I met with Mina regarding other UX research projects that have been undertaken. I also reached out to folks who've done projects with DONE or Neighborhood Councils to see if they did any research projects, and I'm awaiting their answers.

Kiki (UX researcher at Lucky Parking) wants to know more about our impact research and how we can work together.

**Comment by ebele-oputa** [comment]
@ebele-oputa urged Kiki to reach out to Bonnie on admin channel

**Comment by ebele-oputa** [comment]
Check the dependencies for the next steps
